.eslintrc.js

```diff
@@ -43,6 +43,11 @@ module.exports = {
         tsx: 'never',
       },
     ],
+    'wrap-iife': [
+      'error',
+      'inside',
+      { functionPrototypeMethods: true },
+    ],
```

Comment:
This change belongs in a separate, unrelated PR. Let's make an effort to keep our PR history clean.

Reply:
Rebased after the merge of #28